### DISCUSSION TITLE
Welcome to flutterfirstapp Discussions!

### DISCUSSION THREAD

**Discussion by GunvantGMC** [GunvantGMC/flutterfirstapp#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shubham8550** [comment]
cool

**Comment by shubham8550** [comment]
dont you think we should start with setting up actions first
